core/unix/os.c

```diff
@@ -5520,6 +5526,66 @@ was_sigreturn_syscall(dcontext_t *dcontext)
     return is_sigreturn_syscall_number(dcontext->sys_num);
 }
 
+int
+convert_to_non_prace_syscall_number(int sysnum)
+{
+#    if defined(LINUX)
+    if (sysnum == SYS_ppoll) {
+#        if defined(X86) || defined(ARM)
+        return SYS_poll;
+#        else
+        /* XXX: AArch64, no poll, check bionic header for syscall
+         * TEMPORARY, will prob. get removed before commit */
```

Comment:
It was meant to be a temporary scratch comment only, but now since the syscalls seem to be really missing from AArch64, I'll  remove the multi-line temp comment and will replace it with a one liner

Reply:
Done.